plugin/php/customizer/class-controls.php

```diff
@@ -1714,9 +1714,9 @@ public function get_typography_extra_controls( $token ) {
 				'small'  => 2.5,
 			],
 			'title'    => [
-				'large'  => 2,
-				'medium' => 2,
-				'small'  => 1.75,
+				'large'  => 1.33,
+				'medium' => 1.75,
+				'small'  => 1.57,
```

Comment:
1. Default from m3 token where too wide line height for card, 
2. As per design specs of m3 card title should use body large but that is too small.
3. As per m2 to m3 migration this should be title-large, but the line height is too wide so updated default for line-height as per m2 theme specs.